src/account/account.repository.knex.integration.test.ts

```diff
@@ -365,4 +365,50 @@ describe('KnexAccountRepository', () => {
         const follows = await client('follows').select('*');
         expect(follows).toStrictEqual([]);
     });
+
+    it('handles removing follow relationships when a domain is blocked', async () => {
+        // Create Alice from a different domain
+        const [aliceAccount] = await fixtureManager.createInternalAccount(
+            null,
+            'alice-domain.com',
+        );
+
+        // Create Bob from the domain to be blocked
+        const [bobAccount, bobSite] =
+            await fixtureManager.createInternalAccount(
+                null,
+                'blocked-domain.com',
+            );
+
+        // Create Charlie using the same site as Bob
+        const [charlieAccount] =
+            await fixtureManager.createInternalAccount(bobSite);
+
+        // Create follow relationships:
+        // 1. Alice follows Bob (will be removed when domain is blocked)
+        // 2. Charlie follows Alice (will be removed when domain is blocked)
+        await fixtureManager.createFollow(aliceAccount, bobAccount);
+        await fixtureManager.createFollow(charlieAccount, aliceAccount);
+
```

Comment:
Could be good to create follow between charlie and bob and verify thats still there afterwards instead of checking the db is empty?

Reply:
Yeah fair point